tests/network/vmi_slirp_interface.go

```diff
@@ -90,10 +91,17 @@ var _ = SIGDescribe("Slirp Networking", func() {
 		)
 
 		BeforeEach(func() {
-			ports = []v1.Port{{Name: "http", Port: 80}}
+			const dummyIfaceName = "default"
```

Comment:
Curious, why do you call it dummy?

Reply:
You can use `libvmi.DefaultInterfaceName` and avoid this const.

Reply:
> Curious, why do you call it dummy?

No good reason - it was actually misleading.

I've accepted @EdDev 's suggestion and started using the `libvmi` constant.